README.md

```diff
@@ -405,21 +405,25 @@ color: {
 #### Setup
 
 When Using Code from another File (Composition based) or accessing Component Elements via ref, the code needs to be 
-defined in the Setup Method. The setup Method needs to have a proper Return Type by defining an Type:
+defined in the Setup Method. The setup Method needs to have a proper Return Type:
+
 
 
 ```ts
 import { defineComponent, ref, Ref } from 'vue';
-import useFormStates, { IFormStates } from '@/compositions/form-states';
+import useFormStates, { FormStates } from '@/compositions/form-states';
+
+type Setup = FormStates & {
+
 
-type Setup = IFormStates & {
 
```

Comment:
todo: please make sure, this file does not become a mess. Keep empty lines to a minimum.